.github/workflows/ci.yml

```diff
@@ -24,6 +24,10 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v2
+      - name: Build
+        run:  |
+          npm install
+          npm run build
```

Comment:
Remove this step. See my comment about `npm run pre-checkin`

Reply:
Done! https://github.com/crazy-max/ghaction-import-gpg/pull/18/commits/1071df476fb67a0ae68207300af479421a5bae79

Sorry, totally miss that step :(

Reply:
Thanks :)